packages/dd-trace/src/plugins/util/git.js

```diff
@@ -362,6 +362,26 @@ function getGitMetadata (ciMetadata) {
   return tags
 }
 
+function getGitInformationDiscrepancy () {
+  const gitRepositoryUrl = sanitizedExec(
+    'git',
```

Comment:
It may be possible to also read this from the filesystem. It's an entry in `.git/config` though I'm not sure how to tell which is "correct" if there are multiple entries.

In either case reading from the FS should be much faster than shelling out to git. Plus there will be more situations where a git repo is present but the git binary is missing than when there is a git repo present and a git binary present.

Reply:
we have a hard requirement on `git` for a bunch of features. While we could get away with parsing the `.git` folder directly for simple cases, there are things which _need_ the `git` executable. This specific functionality failing because `git` is missing would be non impactful at all compared to the others, so I don't think it makes sense to make this more complicated